ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java

```diff
@@ -168,30 +171,40 @@ public void downloadAndSplit(String taskId, String modelName, String version, St
         }
     }
 
-    private void verifyModelZipFile(String modelZipFilePath) throws IOException {
-        boolean hasModelFile = false;
+    public void verifyModelZipFile(MLModelFormat modelFormat, String modelZipFilePath) throws IOException {
+        boolean hasPtFile = false;
+        boolean hasOnnxFile = false;
         boolean hasTokenizerFile = false;
         try (ZipFile zipFile = new ZipFile(modelZipFilePath)) {
             Enumeration zipEntries = zipFile.entries();
             while (zipEntries.hasMoreElements()) {
                 String fileName = ((ZipEntry) zipEntries.nextElement()).getName();
-                if (fileName.endsWith(PYTORCH_FILE_EXTENSION) || fileName.endsWith(ONNX_FILE_EXTENSION)) {
-                    if (hasModelFile) {
-                        throw new IllegalArgumentException("Find multiple model files, but expected only one");
-                    }
-                    hasModelFile = true;
-                }
+                hasPtFile = hasModelFile(modelFormat, MLModelFormat.TORCH_SCRIPT, PYTORCH_FILE_EXTENSION, hasPtFile, fileName);
```

Comment:
A common PyTorch convention is to save models using either a .pt or .pth file extension. [Ref](https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-model-for-inference)

May be we can extend  `PYTORCH_FILE_EXTENSION` to .pth also?

Reply:
DJL doesn't support .pth